include/aos/common/downloader.hpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_DOWNLOADER_HPP_
+#define AOS_DOWNLOADER_HPP_
+
+#include "aos/common/error.hpp"
+#include "aos/common/string.hpp"
+
+namespace aos {
+namespace downloader {
```

Comment:
All common instances just in aos namespace.

Reply:
Done